templates/components/common/facebook-like-button.html

```diff
@@ -0,0 +1,12 @@
+{{#if settings.facebook_like_button}}
+    {{#if settings.facebook_like_button.enabled}}
```

Comment:
🍹 could switch this to `{{and}}` or just drop the outside `{{if}}`, if the object is not defined handlebars will treat that as falsy.